test/DynamicExpresso.UnitTest/DynamicTest.cs

```diff
@@ -105,6 +106,15 @@ public void Case_Insensitive_Dynamic_Members()
 			Assert.Throws<RuntimeBinderException>(() => interpreter.Eval("dyn.BAR", new Parameter("dyn", dyn)));
 		}
 
+	    [Test]
```

Comment:
There are some minor code alignment problems here and in other files? Probably due to tab/space mismatch ...

Reply:
ok, i will format codes again.

Reply:
Fix [#5e133f3](https://github.com/davideicardi/DynamicExpresso/pull/142/commits/5e133f39e9147002e2fda6b040513fba53b0f598)